The original Makefile did not pass CFLAGS and LDFLAGS
See bug https://bugs.gentoo.org/727632
Also remove -O3 and stripping, leave portage control that
--- a/util/Makefile
+++ b/util/Makefile
@@ -10,10 +10,10 @@
 endif
 
 pdfconcat:
-	$(CC) $(CFLAGS) -O3 -s -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
 
 ppmtops:
-	$(CC) $(CFLAGS) ppmtops.c -o ppmtops
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) ppmtops.c -o ppmtops
 
 ifneq ($(WITH_PGF),no)
 pgf2pnm: main.cpp pnm.cpp
